d2l-typography-shared-styles.html

```diff
@@ -26,6 +26,14 @@
 					text-transform: uppercase;
 				};
 
+				--d2l-body-compact-text: {
+					color: var(--d2l-color-ferrite);
```

Comment:
I'm cool with dropping the color setting from `d2l-body-compact-text` and leaving the others for now. Sound reasonable?